### DISCUSSION TITLE
DuplicateAnker entities necessary (list of entities)

### DISCUSSION THREAD

**Discussion by desvelado** [thomluther/ha-anker-solix#95] [General]
Hello @thomluther 
Thank you for the great Integration and explanation to Automaten output of anker gen1

I habe question regarding the entitied provided by anker. Ist it neccessary to make Duplicates of All entities Provided by anker like Sensors and Switches also as selectors or is it enough to access the original anker entities?

Duplikate Sensors is workifor me but Switches and selector etc. Is not so easy.

Thank you!

**Comment by thomluther** [comment]
@desvelado , not sure what you mean with duplicating the entities?
You can use them as is, but keep in mind that they may become unavailable when something breaks with the integration.
If there is a breaking change, you also may need to adopt your scripts and automation and dashboard with new entitiy names.

The only reason for creating helpers is for using any of the information in the energy dashboard, since switching entities there will cause a loss of energy history data, which is not what you want.

When you create an integral helper, you have already an indirect entity. Just do not use the integration energy entities directly in the dashboard. Most of them are not directly usable anyway, since the energy dashboard is working best with energy entitites reflecting the totals (always increasing), not utility meter type entities as presented by the energy stats entities

**Reply by desvelado** [reply]
I mean this entities i made seperate from the anker integration ones:
sensor.sb_e1600_mode
sensor.sb_e1600_battery_energy
select.sb_e1600_soc_reserve
sensor.sb_e1600_home_preset
switch.sb_e1600_allow_power_export
number.sb_e1600_system_home_preset
sensor.system_bkw_sb_solar_power
sensor.system_bkw_system_home_preset
sensor.system_bkw_sb_state_of_charge
sensor.system_bkw_sb_battery_power

switch & selector & number was not clear for me yesterday how to do it. now i made it with helpers-entities. 

Thanks a lot!

**Reply by thomluther** [reply]
Still don't get your point. Those entities are all integration entities.
The *.sb_e1600_* are device entities, the *.system_bkw_* are system entities from the integration

**Reply by desvelado** [reply]
Sorry i'm a ha newbee ..
May be that makes no sense what i try to do or think that i ve todo:
I Red that some entities like integral counter et need to be generated as a new entities.

I don't know if I could use the integration entities directly or do i need to make a new entity like sensor.my_solarbank_mode.
Dringend that for all entities of the integration and also of the Inverter etc.
